volume/provisioners/jiva/snapshot_revert.go

```diff
@@ -1,28 +1,16 @@
 package jiva
 
 import (
-	"fmt"
-
 	"github.com/openebs/maya/command"
 )
 
 // SnapshotRevert will be responsible for reverting to a
 // particular snapshot. If there is more then one snapshot has been
 // created for a volume, then user can revert to any specific created
 // snaphot for that particular volume.
-func SnapshotRevert(volname string, snapshot string) error {
-
-	annotations, err := command.GetVolumeSpec(volname)
-	if err != nil || annotations == nil {
+func SnapshotRevert(volname string, snapshot string, controllerIP string) error {
```

Comment:
Too many parameters again please encapsulate!

Reply:
Volname and snapshot name can be passed from structure. i will change that.

Reply:
volname is not needed here !

Reply:
done